pkg/unlock.go

```diff
@@ -150,9 +143,7 @@ func runCmdViaDocker(localDirs cliLocalDirectories, command string, extraArgs []
 		"run",
 		"--rm",
 		"-u", currentUser.Uid,
-		"-v", localDirs.secretDir + ":" + localDirs.secretDir,
-		"--env", "HTTP_PROXY=" + os.Getenv("HTTP_PROXY"),
```

Comment:
Don't remove these proxy envs. They are required in some countries to avoid geo restriction.